app/src/main/java/com/github/geohunt/app/model/database/firebase/FirebaseDatabase.kt

```diff
@@ -140,6 +142,62 @@ class FirebaseDatabase(activity: Activity) : Database {
     override fun getNearbyChallenge(location: Location): Task<List<Challenge>> {
         TODO()
```

Comment:
It would be nice to provide a reason for the TODO.

Reply:
It will be implemented soon, this TODO is not meant to last.